### ISSUE TITLE
Add What's New into the app

### ISSUE THREAD

**Issue by rtsisyk** [organicmaps/organicmaps#7017]
<img src="https://github.com/organicmaps/organicmaps/assets/1799054/6d6c2eb8-d662-45d6-aac0-bc86649f7c1c" width="350px">
<img src="https://github.com/organicmaps/organicmaps/assets/1799054/bbbacb2e-c708-49f9-9c1d-9b91cd3c9dd4" width="350px">
<img src="https://github.com/organicmaps/organicmaps/assets/1799054/2eec0131-3080-4d7c-a0d1-b4e51b8b9f48" width="350px">
<img src="https://github.com/organicmaps/organicmaps/assets/1799054/9c03fdce-c26a-4a49-9c5d-0dcd59518742" width="350px">
<img src="https://github.com/organicmaps/organicmaps/assets/1799054/3a9f68fb-eaad-4481-bb8e-728df992f574" width="350px">
<img src="https://github.com/organicmaps/organicmaps/assets/1799054/ec620e9e-06df-4f69-be26-a4b8659ac952" width="350px">

**Comment by meenbeese** [comment]
OM can just pull the news from [https://organicmaps.app/news/](https://organicmaps.app/news/), right? There is no need to re-implement it inside the app.

**Comment by RicoElectrico** [comment]
My idea - people hate popups. So can we do a red dot on the logo button , just like we do on the menu button whenever there are new maps?

**Comment by biodranik** [comment]
Of course, that is the best way. What should happen on the button click after that?
